src/ragger/backend/interface.py

```diff
@@ -42,6 +44,24 @@ def __init__(self, firmware: Firmware):
         self._last_async_response: Optional[RAPDU] = None
         self.raise_policy = RaisePolicy.RAISE_ALL_BUT_0x9000
 
+        self.apdu_logger = logging.getLogger("apdu_logger")
+        self.apdu_logger.setLevel(level=logging.DEBUG)
+        handler = logging.StreamHandler()
+        formatter = logging.Formatter('[%(asctime)s][%(levelname)s] %(name)s - %(message)s')
+        handler.setFormatter(formatter)
+        self.apdu_logger.addHandler(handler)
+
+        if log_apdu_file is not None:
+            apdu_handler = logging.FileHandler(filename=log_apdu_file, mode='w', delay=True)
+            apdu_handler.setFormatter(logging.Formatter('%(message)s'))
+            self.apdu_logger.addHandler(apdu_handler)
+            atexit.register(self._cleanup)
+
+    def _cleanup(self):
+        for handler in self.apdu_logger.handlers:
+            if isinstance(handler, logging.FileHandler):
+                handler.close()
+
```

Comment:
I feel like all this configuration should be in a specific module, rather than buried here (in particular the log path as an argument feels strange to me). There can be a `ragger.logger` module with an `init` function, which would initialize every needed loggers (this one but also the `root` one currently in `ragger.__init__.py`).
These loggers could be initialized explicitly, or implicitly in `ragger.__init__.py` for instance.

```python
import atexit
import logging 
from pathlib import Path
from typing import Optional

def init(log_file: Path, format: Optional[str] = None):
    format = format or '[%(asctime)s][%(levelname)s] %(name)s - %(message)s'

    logger = logging.getLogger(__package__)
    logger.setLevel(level=logging.DEBUG)
    handler = logging.StreamHandler()
    handler.setFormatter(logging.Formatter(format))
    logger.addHandler(handler)

    apdu_logger = logging.getLogger("apdu_logger")
    apdu_logger.setLevel(level=logging.DEBUG)
    handler = logging.StreamHandler()
    handler.setFormatter(logging.Formatter(format))
    apdu_logger.addHandler(handler)

    if log_file is not None:
        apdu_handler = logging.FileHandler(filename=log_file, mode='w', delay=True)
        apdu_handler.setFormatter(logging.Formatter('%(message)s'))
        apdu_logger.addHandler(apdu_handler)
        
       def cleanup():
            for handler in apdu_logger.handlers:
                if isinstance(handler, logging.FileHandler):
                    handler.close()

        atexit.register(cleanup)
```

```python
from ragger import loggers

loggers.init(<args>)
```

Doing so, everybody could now use the loggers everywhere with a `logging.getLogger(<name>)`. That would be cleaner that the current behavior, where `logger` is a global variable fetched with `from ragger import logger`.